procedures/katello/branch.md.erb

```diff
@@ -1,43 +1,83 @@
-* [ ] Add new Katello release job to [foreman-infra](https://github.com/theforeman/foreman-infra/tree/master/puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline)
+* [ ] Ensure stable Pulp release
+  * [ ] Update the [katello-repos](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/katello/katello-repos/katello-repos.spec#L1-L2)
+  * [ ] Update [forklift](https://github.com/theforeman/forklift/blob/master/roles/katello_repositories/defaults/main.yml#L4-L5)
 
-# Main code repos
+## Release Packager
+
+* [ ] Ensure tool_belt config is merged and output from `./tools.rb koji configs/katello/<%= release %>.yaml` matches expectations
+
+# On Branch Date
+
+## Release Owner
 
-* [ ] Ensure releases of installer modules, this is typically done as part of Foreman module release workflow
-    * [ ] [candlepin](https://github.com/theforeman/puppet-candlepin)
-    * [ ] [pulp](https://github.com/theforeman/puppet-pulp)
-    * [ ] [foreman_proxy_content](https://github.com/theforeman/puppet-foreman_proxy_content)
-    * [ ] [katello](https://github.com/theforeman/puppet-katello)
 * [ ] Create KATELLO-<%= release %> branches
   * [ ] [katello](https://github.com/katello/katello)
   * [ ] [katello-installer](https://github.com/katello/katello-installer)
 * [ ] Bump versions to <%= develop %>-master
   * [ ] [katello](https://github.com/katello/katello)
   * [ ] [foreman-installer](https://github.com/katello/katello-installer)
+* [ ] Branch docs
+  * Use `./tools branch-docs config/katello/2.1.yaml` to branch automatically or by hand:
+    * [ ] Copy [nightly manual content](https://github.com/theforeman/theforeman.org/tree/gh-pages/plugins/katello/nightly) to <%= release %>
+    * [ ] Create sidebar from [nightly](https://github.com/theforeman/theforeman.org/tree/gh-pages/_includes/plugins/katello/sidebar_nightly.html) to `sidebar_<%= release %>.html`
+    * [ ] Add <%= release %> to [layout](https://github.com/theforeman/theforeman.org/blob/gh-pages/_layouts/plugins/katello/documentation.html#L3) and set `display` to include `-RC`
+  * [ ] Draft release notes in markdown ([example](https://gist.github.com/domcleal/5567450)), with these sections (and do not use personal pronouns):
+    * [ ] Headline features: half a dozen important features with a few sentences description each
+    * [ ] Upgrade notes: all important notices that users must be aware of before upgrading
+    * [ ] Release notes: bullet point list by category of most changes, excluding bug fixes for issues introduced during the release cycle, include link to bug numbers. You can auto-generate changes using the 'changelog' command in [tool_belt](https://github.com/theforeman/tool_belt)
+* [ ] Update API docs and open a PR with the changes:
 
-# Website
+      VERSION=<%= short_version %>
+      $GITDIR=~/git  #directory containing theforeman.org git repo
```

Comment:
Assigning variables in shell with `$` is usually a mistake

Reply:
True. This was straight copy/paste from the docs. Will update.